finder-app/writer.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/sh
+# Writer script for assignment 1
+# Author: Tim Bailey
+
+NUM_PARAMS=2
+writefile=$1
+writestr=$2
+
+# Check for correct number of input parameters
+if [ $# -ne $NUM_PARAMS ]
+then
+	echo "Incorrect number of parameters. Should be ${PARAMS}, was $#."
```

Comment:
good.. but should this be `NUM_PARAMS`?

Reply:
Certainly should have been; changed it in `finder.sh` and must have overlooked it in `writer.sh`.